_posts/newsletter/next.md

```diff
@@ -5,28 +5,37 @@ permalink: /newsletter/2020-01-07
+
+This is a monthly summary of Solid progress.
+
+Want something mentioned? Contact the Solid team at info@solidproject.org. This Month in Solid is openly developed [on GitHub](https://github.com/solid/solidproject.org/blob/staging/_posts/newsletter/next.md). If you find any errors in this month’s issue, please [submit a pull request](https://github.com/solid/solidproject.org/pulls).
 
 ## Updates
 
 ### [Events](https://solidproject.org/events)
 
 **7 January 2020** - 
-  
-### [Press](https://solidproject.org/press)
+Welcome to the This Month in Solid newsletter! This month was all about reflecting on 2020 and what to expect for the Solid Community in the New Year. We experienced so much growth in various ways from seeing our community numbers go up, to watching servers be released, to learning what developers are doing. As we plan for this year it’s important to see what work needs to be done. 
+
+During January’s Solid World Tim Berners-Lee presented on the high-level overview of Solid. Specifications were the first point made. It was great to hear how much progress the W3C Solid Community Group has made, many thanks to them. We also learned about the numerous servers that are becoming available to developers and organizations. We now have the [Community Solid Server](https://solid.github.io/community-server/docs/) (CSS) and [Inrupt’s Enterprise Solid Server](https://inrupt.com/products/enterprise-solid-server) (ESS), as well as a few others in progress. If we continue on the path we are on now we will get to a place where all applications will truly be interoperable with multiple backend softwares.
+
+There are also some updates to the Solid Operating System (or the SolidOS) [roadmap](https://solidos.solidcommunity.net/public/Roadmap/Tasks/index.ttl#this) to check out. In closing Tim reminded us of the goal of Solid, its community and the ecosystem. It is to build a healthy foundation for Solid so that developers or others who follow can quickly jump into building the type of value they’re interested in. The good news, we are seeing the foundation of that forming. 
+
+Next we heard from Ruben Verborgh, who used the Community Solid Server to demonstrate how his smart lights could be controlled via his Pod. You can see the code behind the demo [here](https://github.com/RubenVerborgh/solid-hue). We also heard from Osmar (Oz) Olivo about Inrupt’s Enterprise Solid Server, reflections on their growth and what they are working toward this year. Last, but very much so not least, we heard from Sarven Capadisli, who spoke about the specification updates on behalf of the W3C Solid Community Group. He demonstrated all the growth they have made to build our current foundation. He also mentioned that the panels for discussing the specs are open to anyone. There were many resources shared during the Solid World, but no worries if you miss them we have the links and resources available [here](https://solidproject.org/events). 
```

Comment:
I think the list of resources is separate from the description of Sarven's talk, right?

```suggestion
Next we heard from Ruben Verborgh, who used the Community Solid Server to demonstrate how his smart lights could be controlled via his Pod. You can see the code behind the demo [here](https://github.com/RubenVerborgh/solid-hue). We also heard from Osmar (Oz) Olivo about Inrupt’s Enterprise Solid Server, reflections on their growth and what they are working toward this year. Last, but very much so not least, we heard from Sarven Capadisli, who spoke about the specification updates on behalf of the W3C Solid Community Group. He demonstrated all the growth they have made to build our current foundation. He also mentioned that the panels for discussing the specs are open to anyone.

There were many resources shared during the Solid World, but no worries if you missed them: we have the links and resources available [here](https://solidproject.org/events). 
```

Reply:
Yea, they are separate

Reply:
Oh also I can create the PR for Jan

Reply:
Actually can you create the Jan newsletter it will be faster that way

Reply:
Will do.